recipes-tag/pystackreg/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pystackreg" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 31324f77136d9666a0c25922edfd38af50d174386cf6661dc2dbcf2e95af67d8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+    - tqdm
+  run:
+    - numpy
+    - python
+    - tqdm
+
+test:
+  imports:
+    - pystackreg
+
+about:
+  home: https://bitbucket.org/glichtner/pystackreg
+  license: UNKNOWN
+  license_family: OTHER
+  license_file: 
```

Comment:
This can probably be replaced with ` license_url: https://bitbucket.org/glichtner/pystackreg/src/97eba915caf6d70618bb825802f43a7057889692/LICENSE.txt?at=master&fileviewer=file-view-default`?